### DISCUSSION TITLE
Intercepting "Automation Vault"

### DISCUSSION THREAD

**Discussion by escequi** [mitmproxy/mitmproxy#6580] [Intercept Everything!]
#### Problem Description
warn: [23:11:55.993][127.0.0.1:59787] Client TLS handshake failed. The client may not trust the proxy's certificate for
automationvault.forumflash.com (OpenSSL Error([('SSL routines', '', 'decryption failed or bad record mac')]))

Have the CA root certs installed;

#### Steps to reproduce the behavior:
1. install mitm proxy
2. install the CA root cert in bin/certi folder
3. start mitm proxy and using proxifier to force traffic through the proxy
4. run the application (C# program, not a browser)

#### System Information
Mitmproxy: 10.1.6 binary
Python:    3.12.0
OpenSSL:   OpenSSL 3.1.4 24 Oct 2023istening at *:8080.
Platform:  Windows-11-10.0.22621-SP0e

**Comment by Prinzhorn** [comment]
The app likely implements [certificate pinning](https://docs.mitmproxy.org/stable/concepts-certificates/#certificate-pinning). Nothing we can do about that.

**Comment by Prinzhorn** [comment]
Although "decryption failed or bad record mac" doesn't sound like pinning? Can you name the app?

**Comment by escequi** [comment]
> Although "decryption failed or bad record mac" doesn't sound like pinning? Can you name the app?

https://automationvault.forumflash.com/forums/ (can't upload file here due to size), feel free to hmu on discord if u need (mr.vit)
